src/nvim/path.c

```diff
@@ -1156,10 +1156,10 @@ int gen_expand_wildcards(int num_pat, char_u **pat, int *num_file,
             ) {
           /* :find completion where 'path' is used.
            * Recursiveness is OK here. */
-          recursive = FALSE;
+          recursive = false;
           add_pat = expand_in_path(&ga, p, flags);
-          recursive = TRUE;
-          did_expand_in_path = TRUE;
+          recursive = true;
+          did_expand_in_path = true;
```

Comment:
lint is complaining for the lack of braces around the else clause just below this line.